src/main/java/core/basesyntax/model/Message.java

```diff
@@ -1,8 +1,22 @@
 package core.basesyntax.model;
 
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "messages")
 public class Message {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany(cascade = {CascadeType.PERSIST, CascadeType.REMOVE})
```

Comment:
- when use toMany association you can get lazyInitializationException, because its' default fetch type is LAZY, let' use EAGER for now to avoid this